java/integTest/src/test/java/glide/standalone/TransactionTests.java

```diff
@@ -0,0 +1,98 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.standalone;
+
+import static glide.TestUtilities.transactionTest;
+import static glide.TestUtilities.transactionTestResult;
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import glide.TestConfiguration;
+import glide.api.RedisClient;
+import glide.api.models.Transaction;
+import glide.api.models.commands.InfoOptions;
+import glide.api.models.configuration.NodeAddress;
+import glide.api.models.configuration.RedisClientConfiguration;
+import java.util.concurrent.TimeUnit;
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class TransactionTests {
+
+    private static RedisClient client = null;
+
+    @BeforeAll
+    @SneakyThrows
+    public static void init() {
+        client =
+                RedisClient.CreateClient(
+                                RedisClientConfiguration.builder()
+                                        .address(
+                                                NodeAddress.builder().port(TestConfiguration.STANDALONE_PORTS[0]).build())
+                                        .build())
+                        .get(10, TimeUnit.SECONDS);
```

Comment:
instead of using `get(10, TimeUnit.SECONDS)` throughout the tests, maybe just add the `@timeout` annotation to the tests? it will reduce the code's verbosity.